tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -86,28 +86,28 @@ def montecarlo_radial1d(model, plasma, runner):
 
     if montecarlo_configuration.VPACKET_LOGGING and number_of_vpackets > 0:
         runner.virt_packet_nus = np.concatenate(
-            np.array(virt_packet_nus)
+            np.array(virt_packet_nus, dtype=object)
```

Comment:
Why are you setting the dtype to object?  These arrays already have the correct dtype

Reply:
Yes but the function `numpy.concatenate` has some updates in the newer version 🤔

Reply:
Which version?

Reply:
![image](https://user-images.githubusercontent.com/66117751/126041012-9afbee31-adca-4e03-ae03-489dc9545d5e.png)
Numpy Version 1.20.0

Reply:
Is that the version specified in the tardis environment yaml?

Reply:
No, its 1.19 in the YAML